hyp3_autorift/process.py

```diff
@@ -180,8 +185,8 @@ def get_product_name(reference_name, secondary_name, orbit_files=None, pixel_spa
 def get_platform(scene: str) -> str:
     if scene.startswith('S1') or scene.startswith('S2'):
         return scene[0:2]
-    elif scene.startswith('L'):
-        return scene[0]
+    elif scene.startswith('L') and scene.startswith(('4', '5', '7', '8'), 3):
```

Comment:
I think this would be more clear:
```suggestion
    elif scene.startswith('L') and scene[3] in ('4', '5', '7', '8'):
```